### PR TITLE
Implemented the toggleable panels on the bottom of the screen

### PR DESCRIPTION
**Pull request by Dacaspex**
#### Deployment instructions
/

#### What have we done
- Implemented the toggleable panels on the bottom of the screen for desktop
- Implemented slideable panels for mobile.
- Implemented menu bar.
- Implemented overflow menu.
- Fixed help page

#### Testing
1. `npm install`
2. Click through the UI on both desktop and mobile
3. Open all panels in different orders
   - Look for inconsistencies

### CODE REVIEW

File: src/app/workspace/mobile-slider-panels/mobile-slider-panels.component.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+    constructor() {
+        // Set first panel visible, the visualisation control
+        this.currentPanel = 0;
+        // Collapse all panels on mobile
+        this.expansionLevel = 0;
+    }
+
+    ngOnInit() {
+        // Intentionally left blank
```
Comment (Chessnl) [inline src/app/workspace/mobile-slider-panels/mobile-slider-panels.component.ts]: Is there a reason why OnInit is implemented, but has no functionality? If this is done on purpose, keep it, else please remove it.

File: src/app/workspace/workspace.component.ts [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
 export class WorkspaceComponent implements OnInit {
 
-  constructor() { }
+    constructor() {
+    }
 
-  ngOnInit() {
-  }
+    ngOnInit() {
+        // Intentionally left blank
```
Comment (Chessnl) [inline src/app/workspace/workspace.component.ts]: Is there a reason why OnInit is implemented, but has no functionality? If this is done on purpose, keep it, else please remove it.

File: src/app/workspace/desktop-toggle-panels/desktop-toggle-panels.component.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    private graphEnabled: boolean;
+
+    constructor() {
+        this.settingsEnabled = false;
+        this.tableEnabled = false;
+        this.graphEnabled = false;
+    }
+
+    ngOnInit() {
+        // Intentionally left blank
```
[the remark this answers is not in the document]
Comment (Dacaspex) [reply src/app/workspace/desktop-toggle-panels/desktop-toggle-panels.component.ts]: It is generated by default by `ng g c <component>`. I would leave it in as that is easier than removing it manually every time.

File: src/app/workspace/mobile-slider-panels/mobile-slider-panels.component.html [hunk 4]
```diff
@@ -0,0 +1,90 @@
+<div
```
[the remark this answers is not in the document]
Comment (Dacaspex) [reply src/app/workspace/mobile-slider-panels/mobile-slider-panels.component.html]: You were right, fixed it

**Pull request merged**